oauthenticator/tests/test_globus.py

```diff
@@ -0,0 +1,35 @@
+from pytest import fixture, mark
+
+from ..globus import GlobusOAuthenticator
+
+from .mocks import setup_oauth_mock, no_code_test
+
+
+def user_model(username):
+    """Return a user model"""
+    return {
+        'login': username,
+    }
+
+@fixture
+def globus_client(client):
+    setup_oauth_mock(client,
+        host=['github.com', 'api.github.com'],
```

Comment:
The SDK helper for this will use `auth.globus.org` internally, so that's what you'll want if you're building this out like other oauthenticators for Jupyterhub.

However, if you're using [`get_authorize_url`](http://globus-sdk-python.readthedocs.io/en/stable/clients/auth/#globus_sdk.AuthClient.oauth2_get_authorize_url), it doesn't really matter what you set here, since the value will never get used.

I'm not sure what the norms of Jupyterhub around this type of library usage are. Should it be `host=None`?